components/button/button.css

```diff
@@ -0,0 +1,90 @@
+.btn-primary {
+
+    background-color: var(--primary-color);
+    padding: 0.8rem;
+    color: var(--primary-white);
+    font-family: var(--primary-font);
+    font-weight: bold;
+    border-radius: 1rem;
+    cursor: pointer;
+}
```

Comment:
border can be set to none so that it looks better, or can be changed to some other color instead of black

Reply:
Thank you for the suggestion!